buildpacks/php/src/layers/platform.rs

```diff
@@ -130,17 +137,19 @@ impl Layer for PlatformLayer<'_> {
                         .ok_or(PlatformLayerError::ProvidedPackagesLogParse)?;
                     // TODO: output filtering and error display (Classic uses echo -n)
                     // TODO: keep in mind that this could, in turn, pull in dependencies
-                    match utils::run_command(composer_require_base.args([
+                    match print::sub_stream_cmd(composer_require_base.args([
                         "require",
                         &format!("{name}.native:*"),
                         // "--no-dev",
                         // "--no-interaction",
                         //"--no-progress",
                     ])) {
-                        Ok(()) => {}
+                        Ok(_) => {}
                         Err(_) => {
                             // TODO: Classic uses \r here
-                            log_info(format!("no suitable native version of {name} available"));
+                            print::sub_bullet(format!(
+                                "no suitable native version of {name} available"
```

Comment:
```suggestion
                                "No suitable native version of {name} available"
```

Seems like it might stick out a bit more that this starts with a lowercase character when it's printed in it's own subbullet?